styles.css

```diff
@@ -59,6 +76,76 @@
   justify-content: space-between;
 }
 
+.show-menu-container {
+  /* background-color: #6070ff; */
+  position: fixed;
+  width: 100%;
+  height: 200px;
+  height: 100%;
+  padding: 20px;
+  mix-blend-mode: multiply;
+  background-color: #6070ff;
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
+}
+
```

Comment:
- [ ] your website looks fantastic I would love to suggest adding a smoother animation to your website's scroll behavior, try to add `scroll-behavior: smooth;` to your HTML tag in CSS it works like magic! 😮 

``` css
html {
scroll-behavior: smooth;
}
```

Before           | After
:-------------------------:|:-------------------------:
![before](https://user-images.githubusercontent.com/89282221/189975888-cd9e04f1-b6a9-4e1e-a5de-bdd05b6d8931.gif) | ![mAFTER](https://user-images.githubusercontent.com/89282221/189975757-65de4564-3fc4-40b9-b40e-1bf3919546b8.gif)

Reply:
Thank you @ichala I will add the suggestion before merging.